doc/widgets/moving_transform_w.md

```diff
@@ -13,26 +13,51 @@ Apply rolling window functions to the time series. Use this widget to get a seri
 
 In this widget, you define what aggregation functions to run over the time series and with what window sizes.
 
-![](images/moving-transform-stamped.png)
+![](images/Moving-Transform.png)
 
-1. Define a new transformation.
-2. Remove the selected transformation.
-3. Time series you want to run the transformation over.
-4. Desired window size.
-5. Aggregation function to aggregate the values in the window with. Options are: *mean*, *sum*, *max*, *min*, *median*, *mode*, *standard deviation*, *variance*, *product*, *linearly-weighted moving average*, *exponential moving average*, *harmonic mean*, *geometric mean*, *non-zero count*, *cumulative sum*, and *cumulative product*.
-6. Select *Non-overlapping windows* options if you don't want the moving windows to overlap but instead be placed side-to-side with zero intersection.
-7. In the case of non-overlapping windows, define the fixed window width(overrides and widths set in (4).
+1. Define the aggregation type:
+   - *Sliding window*: data will be aggregated with the sliding window of the specified width. The function is typically used for smoothing the data.
+      - *Window width*: the number of instances used for creating aggregation
+      - define data output: *Discard original data* will output only aggregated columns, *Keep original data* will output both the original columns and the aggregated columns, *Include leading instances* will output original and aggregated column, including the leading instances that fall outside of the window
```

Comment:
"Falling outside the window" sounds as if these are the last instances.

Maybe:

*Include leading instances* will output original and aggregated column, including the first N - 1 leading instances (where N is the window width) before the first complete window. The values in aggregated columns for these instances are left undefined.